src/main/mule/password-manager-api.xml

```diff
@@ -1,5 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd"></mule>
+<mule xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd">
+	<flow name="password-manager-api-main" doc:id="552b5356-b2e3-494a-9c5b-3bbd177958bb" >
+		<!-- <http:listener doc:name="Listener" doc:id="e38920ae-9809-4c37-aeae-185d5dcbfe4c" config-ref="${secure::password-manager-api.anypoint.listener.config-ref}" path="${secure::password-manager-api.anypoint.base-path}"> -->
+		<http:listener doc:name="Listener" doc:id="7525f939-018f-4f9b-8850-44f4f64d427c" config-ref="password-manager-api-https" path="${secure::password-manager-api.anypoint.base-path}">
```

Comment:
To be replaced: hardcoded value of listener config
To be deleted: comment at line 8